node_modules/gh-admins/tests/util.js

```diff
@@ -270,6 +270,66 @@ var assertUpdateGlobalAdminFails = module.exports.assertUpdateGlobalAdminFails =
 };
 
 /**
+ * Assert that a global administrator's password can be changed
+ *
+ * @param  {RestClient}         client                          The REST client to make the request with
+ * @param  {Number}             id                              The id of the global administrator for which to change the local password
+ * @param  {String}             newPassword                     The new password for the global administrator
+ * @param  {Function}           callback                        Standard callback function
+ * @param  {RestClient}         callback.client                 A client that's authenticated with the new password
+ * @throws {AssertionError}                                     Error thrown when an assertion failed
+ */
+var assertChangePassword = module.exports.assertChangePassword = function(client, id, newPassword, callback) {
+    // Change the global administrator's password
+    client.admin.changePassword(id, newPassword, function(err) {
+        assert.ok(!err);
+
+        // Grab the admin's profile information so we can attempt a login
+        // and verify the password was changed successfully
+        assertGetAllGlobalAdmins(client, null, function(globalAdmins) {
+            var globalAdmin = _.findWhere(globalAdmins, {'id': id});
```

Comment:
Can you create an `assertGetGlobalAdmin` method that covers getting all and finding? I'm guessing it should return a `false`y value instead of an error code if the specified admin doesn't exist, just due to the semantics.